internal/rukpak/preflights/crdupgradesafety/crdupgradesafety.go

```diff
@@ -96,6 +101,11 @@ func (p *Preflight) Upgrade(ctx context.Context, rel *release.Release) error {
 
 		oldCrd, err := p.crdClient.Get(ctx, newCrd.Name, metav1.GetOptions{})
 		if err != nil {
+			// if there is no existing CRD, there is nothing to break
+			// so it is immediately successful.
+			if apierrors.IsNotFound(err) {
+				return nil
```

Comment:
Should this be `continue`?

Reply:
Ah, you are right - good catch. For some reason I missed that this was in a for loop 🤦

Reply:
Switched to continue in https://github.com/operator-framework/operator-controller/pull/1104/commits/233d5f641e1e8056d052d888a195ed7ce5104d85